package.json

```diff
@@ -21,12 +21,13 @@
     "@babel/core": "^7.4.4",
     "@babel/preset-env": "^7.4.4",
     "babel-loader": "^8.0.5",
+    "babel-polyfill": "^6.26.0",
     "jest": "^24.7.1",
     "simsala": "^0.0.12",
     "webpack": "^4.31.0",
     "webpack-cli": "^3.3.2"
   },
   "dependencies": {
-    "axios": "^0.19.0"
+    "isomorphic-fetch": "^2.2.1"
```

Comment:
axios is usable across environments, why do you want to switch to isomorphic-fetch?

Reply:
On a second thought, this makes the repo cleaner. 👍

Reply:
Didn't see it being used anywhere, the API you're using is the `fetch` API, not axios.

Reply:
Just noticed that aswell! Thanks for catching it.